Patches/JDS Exiled Dawn/Thingdefs_Misc/Apparel.xml

```diff
@@ -0,0 +1,428 @@
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[@Name="JDSExiledScholarArmorBase"]/statBases/Mass</xpath>
+				<value>
+					<Mass>50</Mass>
+					<Bulk>100</Bulk>
+					<WornBulk>25</WornBulk>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[@Name="JDSExiledScholarArmorBase"]/equippedStatOffsets/MoveSpeed</xpath>
+				<value>
+					<CarryBulk>10</CarryBulk>
+					<CarryWeight>60</CarryWeight>
+				</value>
+			</li>
+			
+			<!--Helmets-->
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="ExiledDawn_Casque_Helmet"]/statBases/ArmorRating_Sharp</xpath>
+				<value>
+					<ArmorRating_Sharp>7</ArmorRating_Sharp>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="ExiledDawn_Casque_Helmet"]/statBases/ArmorRating_Blunt</xpath>
+				<value>
+					<ArmorRating_Blunt>17.5</ArmorRating_Blunt>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="ExiledDawn_Casque_Helmet"]/statBases</xpath>
+				<value>
+					<Bulk>5</Bulk>
+					<WornBulk>2</WornBulk>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="ExiledDawn_FootSoldier_Headgear" or
+				defName="ExiledDawn_Defender_Helmet" or
+				defName="ExiledDawn_Headguard"
+				]/statBases/ArmorRating_Sharp</xpath>
+				<value>
+					<ArmorRating_Sharp>9</ArmorRating_Sharp>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="ExiledDawn_FootSoldier_Headgear" or
+				defName="ExiledDawn_Defender_Helmet" or
+				defName="ExiledDawn_Headguard"
+				]/statBases/ArmorRating_Blunt</xpath>
+				<value>
+					<ArmorRating_Blunt>22.5</ArmorRating_Blunt>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="ExiledDawn_FootSoldier_Headgear" or
+				defName="ExiledDawn_Defender_Helmet" or
+				defName="ExiledDawn_Headguard"
+				]/statBases</xpath>
+				<value>
+					<Bulk>5</Bulk>
+					<WornBulk>3</WornBulk>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="ExiledDawn_Knight_Headgear"]/statBases/ArmorRating_Sharp</xpath>
+				<value>
+					<ArmorRating_Sharp>19.8</ArmorRating_Sharp>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="ExiledDawn_Knight_Headgear"]/statBases/ArmorRating_Blunt</xpath>
+				<value>
+					<ArmorRating_Blunt>49.5</ArmorRating_Blunt>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="ExiledDawn_Knight_Headgear"]/statBases</xpath>
+				<value>
+					<Bulk>5</Bulk>
+					<WornBulk>3</WornBulk>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="ExiledDawn_Knight_Headgear"]/equippedStatOffsets</xpath>
+				<value>
+					<SmokeSensitivity>-0.98</SmokeSensitivity>

```

Comment:
Why isn't this just `-1`?

Reply:
I was matching the toxic sensitivity, but there isn't any reason not to just set it to -1. Fixed